internal/server/handlers_certificate_requests.go

```
@@ -346,7 +346,7 @@ func DeleteCertificate(env *HandlerConfig) http.HandlerFunc {
 }
 
 // RevokeCertificate handler receives an id as a path parameter,
-// and attempts to revoke the corresponding certificate request
+// and attempts to revoke the corresponding certificate request by adding the certificate to the CRL
 // It returns a 200 OK on success
 func RevokeCertificate(env *HandlerConfig) http.HandlerFunc {
```

Notary won't be able to revoke a certificate that was signed outside of it - we should probably return a specific error when someone tries to do this.

---

added the same message as [my comment above](https://github.com/canonical/notary/pull/204#discussion_r2026397449).